script.js

```diff
@@ -0,0 +1,226 @@
+// Header Nav Items
+document.getElementById("home").focus();
+
+let tweetBtn = document.getElementById("tweet-btn");
+
+// Enable Tweet Btn
+document.getElementById("tweet-content").addEventListener('keypress', function(){
+    tweetBtn.setAttribute("aria-disabled", "false");
```

Comment:
What if I tweet my first tweet, and the text area becomes empty, shouldn't the 'button' disable?
I mean it depends on what text area content (empty or not)  not if keypress or not!

Reply:
resolved

Reply:
Also, I added media queries